packages/database/src/migrations/20210210003915-AddFETPGradAreaOfExpertiseOverlays-modifies-data.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NEW_OVERLAY_GROUP_ID = generateId();
+
+// this contains shared properties across all overlays
+const BASE_OVERLAY = {
+  userGroup: 'FETP Graduates',
+  isDataRegional: true,
+  measureBuilderConfig: {
+    aggregationType: 'SUM',
+    entityAggregation: {
+      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
+      dataSourceEntityType: 'individual',
+      aggregationEntityType: 'district',
```

Comment:
I think you can also change this to below and it still gives the same result! (but no need to change it)
```suggestion
    entityAggregation: {
      aggregationType: 'SUM_PER_ORG_GROUP',
      dataSourceEntityType: 'individual',
      aggregationEntityType: 'district',
```